playbooks/archivematica-bionic/ansible.cfg

```diff
@@ -1,2 +1,3 @@
 [defaults]
 nocows = 1
+pipelining = True
```

Comment:
Hi @sevein 

The `pipelining` option must be used on `[ssh_connection]` section, like this:
```
[defaults]
nocows = 1

[ssh_connection]
pipelining = True
```

Reply:
Done! Confirmed with `ansible-config dump` too.